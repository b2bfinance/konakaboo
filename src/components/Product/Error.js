import React from 'react';
import styled from 'styled-components';

export const Wrapper = styled.div`
  background-color: #ff9800;
  color: #fff;
  padding: 1rem;
`;

export default ({ message }) => <Wrapper>{message}</Wrapper>;
