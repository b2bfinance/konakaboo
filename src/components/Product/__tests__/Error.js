import React from 'react';
import { render } from 'enzyme';
import Error from '../Error';

describe('Error', () => {
  test('renders correctly', () => {
    expect(render(<Error />)).toMatchSnapshot();
  });
});
