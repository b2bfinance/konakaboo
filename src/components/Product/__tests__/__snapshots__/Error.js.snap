// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error renders correctly 1`] = `
.c0 {
  background-color: #ff9800;
  color: #fff;
  padding: 1rem;
}

<div
  class="c0"
/>
`;
